src/components/Books.js

```diff
@@ -0,0 +1,40 @@
+import Book from './Book';
+import Form from './Form';
+
+const Books = () => {
+  const books = [
+    {
+      id: 1,
+      title: 'The Hunger Games',
+      category: 'Action',
+      author: 'Suzanne Collins',
+      completed: '64',
+      chapter: 'Chapter 3',
+    },
+    {
+      id: 2,
+      title: 'Dune',
+      category: 'Science Fiction',
+      author: 'Frank Herbert',
+      completed: '8',
+      chapter: 'Chapter 17: A lesson learned',
+    },
+    {
+      id: 3,
+      title: 'Capital in the Twenty-First Century',
+      category: 'Economy',
+      author: 'Andrei Tamaioaga',
+      completed: '90',
+      chapter: 'A final word',
+    },
+  ];
+
+  return (
+    <main className="hero">
+      <Book books={books} />
+      <Form />
+    </main>
```

Comment:
- [ ] Let's refactor a little bit the component Book and Books. Since Book is a singular component, I think it should only render one book at a time. For this, we will pass to this component just the props that are pertinent to it, such as id, title, category, author, completed, and chapter.

We will map in the component Books instead (Optional, you could rename this one to BookList to maybe avoid confusion) and render the Book component for each book.

This will hopefully be more coherent and will avoid confusion in the future. :sweat_smile:

Reply:
Hiya @D4nielJ,

I must say that this suggestion finds me a little bit confused.
I believe I am passing already only the props that are pertinent to it, such as id, title, category, author, completed, and chapter. I deleted that prop which was related to deleting books since I'm not using it in this mock part of the project.

And how exactly should I still send props to the Book file and use the props without mapping?
And again, how should I map the BookShelf?

Please, at the same time, remember that this stage of the project is yet to be worked upon.

Thanks a lot for your help and your understanding!

Regards, 
Nicu

Reply:
Thanks a lot @D4nielJ, for helping me refactor using best practices.